draft-ietf-rtcweb-jsep.xml

```diff
@@ -1600,7 +1600,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>The next step is to generate session-level lip sync groups as
           defined in <xref target="RFC5888"/>, Section 7.  For each MediaStream
-          with more than one MediaStreamTrack, a group of type "LS" MUST be
+          with more than one MediaStreamTrack, a group of type LS MUST be
```

Comment:
we used quotes in the preceding paragraph, so I think we should keep them here too. We may want to say "with semantics "LS"" instead of "of type LS" as well.